rollup.config.js

```diff
@@ -42,9 +42,10 @@ function getConfig(input, file) {
 }
 
 const config = [
-  getConfig('./src/index.ts', 'dist/redux-zero'),
-  getConfig('./src/react/index.ts', 'react/index'),
-  getConfig('./src/svelte/index.ts', 'svelte/index'),
+  getConfig("./src/index.ts", "dist/redux-zero"),
+  getConfig("./src/react/index.ts", "react/index"),
+  // getConfig("./src/preact/index.ts", "preact/index"),
```

Comment:
Uncomment this, right?

Reply:
that's the idea for when everything is working. I'm keeping as it is for now